BotHelperMethods/AdaptiveCards/WelcomeCard.json

```diff
@@ -81,13 +81,7 @@
     {
       "type": "Action.Submit",
       "title": "%TakeATourButtonText%",
-      "data": {
```

Comment:
@suramn I'd say that you want to have this data:{....} JSON object included because at the end of the day, the FAQ++ bot will be running in Teams